### ISSUE TITLE
Merge Basic Details and Additional Details

### ISSUE THREAD

**Issue by tinashe404** [atsign-company/at_wavi_app#146]
**Is your feature request related to a problem? Please describe.**
Currently in the app, Basic Details contains `First Name, Last Name, Phone and Email` and Additional Detail Details contains `Pronoun and About`. When looking at the page on wavi.ng, this information is all displayed under one section. The app allows people to re-order the information they enter in the app however because these two sections are separated, you cannot reorder information from these two sections in parallel, eg you cannot have Phone followed by Pronoun then Email. 

**Describe the solution you'd like**
It would be great if the Basic Details and the Additional Details sections could be combined into one section that still can have custom content added to it. That way, people would be able to re-order their page without the limitation mentioned above.

**Comment by keyurgit45** [comment]
Hi, I am Keyur Gandhi, a contributor to OpenForce 2022. I would like to work on this issue. I would be making a PR as soon as I am done with resolving the issue. Thank you

**Comment by Ninad07** [comment]
Hi @tinashe404. Does this mean the two rows `Basic Details` and `Additional Details` are supposed to be merged into a single row?
